### ISSUE TITLE
Error: Can't resolve 'react/lib/Object.assign' with React 15.0

### ISSUE THREAD

**Issue by nil4** [KyleAMathews/react-spinkit#28]
Using react-spinkit version 1.1.4 with [React 15.0](https://facebook.github.io/react/blog/2016/04/07/react-v15.html) results in the following error:

> ModuleNotFoundError: Module not found: Error: Can't resolve 'react/lib/Object.assign' in 'C:\projects\MyProject\node_modules\react-spinkit\dist'

**Comment by KyleAMathews** [comment]
So that require wasn't even being used. Removed it and published 1.1.5. https://github.com/KyleAMathews/react-spinkit/commit/1404b2702e61ad2416a95afacefe20732628b60b

**Comment by nil4** [comment]
@KyleAMathews awesome, thank you for the fantastic turn-around time!

**Comment by nil4** [comment]
@KyleAMathews I upgraded to 1.1.5 but still fails; `dist/index.js` still has the line:

```js
objectAssign = require('react/lib/Object.assign');
```

**Comment by KyleAMathews** [comment]
Ooops! Forgot to compile the src. I re-published 1.1.6.

**Comment by nil4** [comment]
@KyleAMathews thanks a lot! Unfortunately there's another issue: [`React.__spread` has been removed](https://github.com/facebook/react/pull/6431) in v15.0, and the `<div {... @props}` syntax seems to compile down to it.

**Comment by KyleAMathews** [comment]
Ha :-) would you mind taking a crack at solving this?

**Comment by KyleAMathews** [comment]
https://github.com/facebook/react/pull/6444

**Comment by nil4** [comment]
Tracked it down to `cjsx-loader`, specifically this: https://github.com/jsdf/coffee-react-transform/pull/32. I'll open an issue on there. For now, I can unblock myself by using just the CSS styles; it would be a shame though, since react-spinkit is such a nice library -- thank you again.

**Comment by KyleAMathews** [comment]
If you see the above PR, they're adding it back in.

**Comment by zpao** [comment]
You should still stop using it :)

**Comment by KyleAMathews** [comment]
This was written back in my hellion Coffeescript days ;-)

**Comment by nil4** [comment]
I opened https://github.com/jsdf/coffee-react-transform/issues/70 while waiting for React 15.0.1 :smile:.

**Comment by KyleAMathews** [comment]
15.0.1 was released.

**Comment by nil4** [comment]
@KyleAMathews: jsdf/coffee-react-transform#70 was fixed, removing the use of `React.__spread`. Would you consider upgrading to version 4.0 to fix this one for good?

**Comment by KyleAMathews** [comment]
Updated and released 1.1.7.
